src/components/picker/picker.tsx

```diff
@@ -273,17 +305,74 @@ export class Picker {
      * @returns {HTMLElement} picker dropdown
      */
     private renderDropdown() {
-        const content = this.getDropdownContent();
+        const dropDownContent = this.getDropdownContent();
+
+        const content = [];
+
+        if (this.shouldShowDropDownContent()) {
+            const actionContent = this.getActionContent();
+            if (this.actionPosition === 'top') {
+                content.push(actionContent);
+            }
+
+            if (dropDownContent) {
+                content.push(dropDownContent);
+            }
+
+            if (this.actionPosition === 'bottom') {
+                content.push(actionContent);
+            }
+        }
 
         return this.renderPortal(content);
     }
 
-    private getDropdownContent() {
+    private getActionContent() {
+        const actionCount = this.actions?.length ?? 0;
+        if (actionCount === 0) {
+            return null;
+        }
+
+        return [
+            <limel-list
```

Comment:
I think we need to figure out some kind of solution to this in the longer run, but for now, since the default is that the static actions end up at the bottom, adding actions with that default behaviour to an existing list doesn't break any of the existing behaviour; the user simply won't be able to access the _new_ functionality using the keyboard alone, but there should probably always be a different way to access that functionality anyway. In the case of inline creation of new objects, you can always go create a new object the old fashioned way instead. So I don't think this is a merge blocker 👍

Reply:
So maybe create an issue and merge this?

Reply:
I was working on it, but Friday demos interrupted me… 😅